README.md

```diff
@@ -1,2 +1,48 @@
+ 
+:speech_balloon: Embedded Chat :speech_balloon:
+===============================================
+<img align="left" width=207 style="margin-right: 20px" src="https://user-images.githubusercontent.com/52821952/64411882-dcd30700-d096-11e9-9460-9deff5e4c14c.png">
+Pair Storm IDE has embedded chat. It helps to communicate with another programmers during development process. In chat can hang out many users at the same time. All messages are saving in local database, which is unique for each project. Embedded chat is secure. Get access to the message history can only authorized users, which work have appropriate premissions.
+
+:hammer: Refactoring Tools :hammer:
+===================================
+<img align="right" width=220 style="margin-left: 20px" src="https://user-images.githubusercontent.com/52821952/64385117-dc0e8680-d03e-11e9-993a-bc40d504a335.png">
+IDE gives users many useful tools, which make development process easier and faster. For example, Pair Storm has embedded class generator. Using it programmer can create .h and .cpp files, define new methods in both file and etc. 
+
+Also Pair Storm have many useful features, which help programmer to write good code: auto tabulation, auto completion and etc.
+
+:high_brightness: Syntax Highlighting :high_brightness:
+=======================================================
+<img align="left" width=500 style="margin-right: 20px" src="https://user-images.githubusercontent.com/52821952/64383380-70c4b480-d03e-11e9-8eed-fe0789acc060.png">
+Embedded code editor displays text in different colors. This helps to make source code more readable, simple for comprehension. Also highlighter recognize invalid construction. This lets find mistakes errors in source code and fix them before compilation.
+
+
+:heavy_plus_sign: Comments :heavy_plus_sign:
+============================================
+<img align="right" width=480 style="margin-left: 20px" src="https://user-images.githubusercontent.com/52821952/64407659-8e6d3a80-d08d-11e9-94df-462c6e6eda4c.png">
+Pair Storm supports comments in Github style. This helps to realize conception of pair programing, when one person write source code and another - review it right away. All comments are saving in local database, which is unique for each project. User can write comments to either line of code.
+
+:wrench: Settings :wrench:
+==========================
+<img align="left" width=500 style="margin-right: 20px" src="https://user-images.githubusercontent.com/52821952/64408625-e3aa4b80-d08f-11e9-89f2-3d5e8d0a0bd4.png">
+With the help of settings menu user can customize IDE in conformity with its own wishes. For example, Pair Storm supports couple styles. Using settings window user can switch between them.
+
```

Comment:
If user wants to customize Pair Storm, we propose ready themes with predefined palettes in the settings menu. Also, for more advanced users, the one can uniformly customize IDE in conformity with his/her own wishes in the settings advanced tabs of the settings menu.